sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -38,9 +38,9 @@ public async Task CreateAndGetAlertConfigurationWithWholeSeriesScope(bool useTok
 
             await using var disposableConfig = await DisposableAlertConfiguration.CreateAlertConfigurationAsync(adminClient, configToCreate);
 
-            AnomalyAlertConfiguration createdConfig = await adminClient.GetAlertConfigurationAsync(disposableConfig.Id);
+            AnomalyAlertConfiguration createdConfig = disposableConfig.Configuration;
 
-            Assert.That(createdConfig.Id, Is.EqualTo(disposableConfig.Id));
+            Assert.That(createdConfig.Id, Is.Not.Null.And.Not.Empty);
```

Comment:
Same reasoning here as we did for `DisposableDataFeed`. `Create` operations don't really have something to validate the ID with, so we're just checking if an ID is being returned. `Update` operations can compare the ID with the ID we had before the update.